.mise.toml

```diff
@@ -0,0 +1,3 @@
+[tools]
+node = "lts-iron"
```

Comment:
mise will use `.nvmrc` if not present, so I think we can just remove this and keep the version in one place?